### ISSUE TITLE
Android Studio 3.1 Build Fails

### ISSUE THREAD

**Issue by allenbaker** [algolia/instantsearch-android-examples#8]
Cloned then opened the Media project using Android Studio 3.1.

Gradle project sync fails with:
```Unable to resolve dependency for ':custombackend@debug/compileClasspath': Could not resolve project :instantsearch.
Unable to resolve dependency for ':custombackend@debug/compileClasspath': Could not resolve project :instantsearch.
Unable to resolve dependency for ':custombackend@debugUnitTest/compileClasspath': Could not resolve project :instantsearch.
Unable to resolve dependency for ':custombackend@release/compileClasspath': Could not resolve project :instantsearch.
Unable to resolve dependency for ':custombackend@releaseUnitTest/compileClasspath': Could not resolve project :instantsearch.```

Edit: Commit 0678b appears to be the source of the issue

**Comment by robertmogos** [comment]
Hi @allenbaker. Sorry for the issue. Indeed, we have some problems with the latest deployment. To unblock your problem just checkout the previous commit `d878aeb60277a63a01e2ea233ef1cfb12af25c1e`. This should fix it for now

**Comment by PLNech** [comment]
Hi @allenbaker, the issue was resolved in release v1.7.25. This repository has been updated and I'm now able to sync Gradle and build any subproject.

Should the issue persist, please reopen this issue and I'll investigate further!

**Comment by PLNech** [comment]
Hi @allenbaker, the issue was resolved in [release v1.7.24](https://github.com/algolia/instantsearch-android/releases/tag/1.7.24). This repository has been updated and I'm now able to sync Gradle and build any subproject.

Should the issue persist, please reopen this issue and I'll investigate further!
